spec/mailers/prompt_mailer_spec.rb

```diff
@@ -53,6 +53,19 @@
         end
       end
 
+      it "describes when the user created their entry" do
+        Timecop.freeze(Date.new(2014, 1, 1)) do
+          user = create(:user)
+          entry = create(:entry, user: user, date: Date.new(2014, 1, 3))
```

Comment:
It appears as though we're creating an entry in the future or am I reading this wrong? :rocket: